DATAFLOW.md

```diff
@@ -1,28 +1,27 @@
 # Enterprise Knowledge Solution (EKS) Dataflow
 
 1. **Upload documents and trigger document processing workflow**
-   An employee (IT) manually uploads various document types (Excel, PDF, Word, Text, Outlook, .xlsx, .pdf, .docx) or a scheduled Cron Job triggers the upload. The uploaded files are stored in a designated "input" bucket within Google Cloud Storage.
-
-   The upload to the "input" bucket triggers a Document Processing Workflow managed by Cloud Composer.
-
+   An employee (IT) manually uploads various document types (Excel, PDF, Word, Text, Outlook) or a scheduled Cron Job triggers the upload. The uploaded files are stored in a designated "input" bucket within Google Cloud Storage.
+   
+   The upload to the "input" bucket triggers a Document Processing Workflow managed by Cloud Composer.  
 1. **Validate the documents:**
    Documents that fail validation are redirected to a separate "reject" bucket within Google Cloud Storage for further review and handling.
```

Comment:
I don't think we ever explained the criteria for how documents are rejected. Is it unsupported file types?

Reply:
I'm not sure if we explained it or not, but if not, we definitely should. Unsupported file types is one, but we also reject files that are too big, or files that we detected as duplicates.

Reply:
We are not rejecting the duplicates at the moment.